### ISSUE TITLE
[Question] 1.20 ?

### ISSUE THREAD

**Issue by HowardZHY** [jellysquid3/phosphor-fabric#95]
Will it update to 1.20?

**Comment by TheUserCreated** [comment]
1.20 dramatically improved the performance of the lighting engine. There hasn't been an official yes or no as to whether phosphor will be updating, but I wouldn't bet on it at this time

**Comment by PweSol** [comment]
1.20 vanilla is basically a copy of [Starlight](https://github.com/PaperMC/Starlight). Phosphor is technically inferior so there is no point updating it.

**Comment by 64BitUniverse** [comment]
Who knows, there could still be improvements to be made somewhere

**Comment by 42Willow** [comment]
I'm not sure if this is to do with not having phosphor, but my game seems a lot more laggy on 1.20 in the nether (where there is lots of light updates) - it's probably partially to do with me having no lighting engine optimisation.

**Comment by NotLicense14** [comment]
I would love to see Phosphor be updated to 1.20[.1],

**Comment by mrjasonn** [comment]
@HowardZHY Phosphor is not needed as Minecraft 1.20 hugely optimised the lighting engine. It’s still not as fast as Starlight, but it is much faster and much closer to Starlight speeds because it has more checks for stability purposes (which Minecraft being bug free is important!)

**Comment by HowardZHY** [comment]
> @HowardZHY Phosphor is not needed as Minecraft 1.20 hugely optimised the lighting engine. It’s still not as fast as Starlight, but it is much faster and much closer to Starlight speeds because it has more checks for stability purposes (which Minecraft being bug free is important!)

I just wonder if there could still be improvements to be made somewhere

**Comment by astridellensdottir** [comment]
Maybe you can find those possible improvements and submit a PR, then. There’s no need for phosphor to update past 1.19 right now, as the lighting engine it improved has been entirely replaced in 1.20 vanilla with one that’s significantly faster than 1.19 with phosphor, and more stable than 1.19 with starlight. The old optimizations no longer apply, and new ones haven’t yet been found.

**Comment by 42willow** [comment]
https://gist.github.com/Spottedleaf/6cc1acdd03a9b7ac34699bf5e8f1b85c

See here

**Comment by jellysquid3** [comment]
We don't know if there will be a port of MC 1.20. Right now, most of Phosphor's patches are rendered broken and/or obsolete. It remains to be seen if there are any significant optimizations which exist for the newer lighting engine that are useful outside of very large servers.

I wouldn't expect a "port" any time soon. If there is an update, it would likely look very different compared to earlier releases.

**Comment by 13Ralef** [comment]
> Will it update to 1.20?

No, because 1.20 has improved it's lightning engine performance dramatically, so phosphor it not necessary anymore I'm those versions, Neither does Starlight.

**Comment by GameLord2011** [comment]
Note: as of today, the Phosophor mod has been officially discontinued

**Comment by douira** [comment]
Where did you get that news from

**Comment by djmrFunnyMan** [comment]
How have I been sleeping on this information for so long?

**Comment by jellysquid3** [comment]
Phosphor has not been maintained for years and our project pages on CurseForge and Modrinth reflect this status.
